src/main/java/emu/grasscutter/game/entity/gadget/platform/BaseRoute.java

```diff
@@ -24,8 +25,8 @@ public abstract class BaseRoute {
 
     BaseRoute(SceneGadget gadget) {
         this.startRot = new Position(gadget.getRot());
-        this.isStarted = gadget.isStartRoute();
-        this.isActive = gadget.isStartRoute();
+        this.isStarted = false;
```

Comment:
you should probably not fully ignore this script variable

Reply:
Fixed in 5a2b49319bdd4edc3dc802c22e8e04d1e98a6e55